src/client/main.tsx

```diff
@@ -24,22 +26,45 @@ export class Main extends React.Component<{}, ClientState> {
       stage: Stages.SPLASH,
       numPlayersInLobby: 0,
     };
+    let pathName = window.location.pathname.substring(1); // Cutting off preceding '/'
+    if (pathName.length === GAME_CODE_LENGTH) {
```

Comment:
If we have other URLs then we should separate between the two with some other form, like having the path be `/game/12345` vs `/othersection/blah`.  `GAME_CODE_LENGTH` is a bit of a flimsy way to decide when we want a game id and when we don't.